README.md

```diff
@@ -117,7 +117,7 @@ They are found in the [`benchmarks/`](https://github.com/unitaryfund/mitiq/tree/
 
 If you use Mitiq in your research, please reference the [Mitiq whitepaper](https://quantum-journal.org/papers/q-2022-08-11-774/) using the bibtex entry found in [`CITATION.bib`](https://github.com/unitaryfund/mitiq/blob/master/CITATION.bib).
 
-A list of papers citing Mitiq can be found on [Google Scholar](https://scholar.google.com/scholar?cites=12810395086731011605) / [Semantic Scholar](https://www.semanticscholar.org/paper/Mitiq%3A-A-software-package-for-error-mitigation-on-LaRose-Mari/dc55b366d5b2212c6df8cd5c0bf05bab13104bd7#citing-papers).
+A list of papers citing Mitiq can be found on [Google Scholar](https://scholar.google.com/scholar?cites=12810395086731011605) / [Semantic Scholar](https://www.semanticscholar.org/paper/Mitiq%3A-A-software-package-for-error-mitigation-on-LaRose-Mari/267e06208dff1060078ceccdd927a2c7c95e86d9#citing-papers).
```

Comment:
```suggestion
A list of papers citing Mitiq can be found on [Google Scholar](https://scholar.google.com/scholar?cites=12810395086731011605) / [Semantic Scholar](https://api.semanticscholar.org/CorpusID:221555755?utm_source=wikipedia).
```

Reply:
@purva-thakre happy to use the API link, but any reason to use it over the one Min used? Maybe it's more stable?

Reply:
When I checked `docs` last night, there was an error with the link Min used. Which is why i was thinking API link could be more stable.

Reply:
Ah yeah it does look like https://github.com/unitaryfund/mitiq/pull/1912/commits/56d4c6583b7120c913dd93f91876dfc79d37c922 failed with an error related to the `citing-papers` anchor (which is weird because the page does have a `citing-papers` anchor). I suggest we use the link Purva suggested (with the `utm_source` query parameter removed.